kubelet/metadata.csv

```diff
@@ -86,6 +86,8 @@ kubernetes.liveness_probe.success.total,count,,,,Cumulative number of successful
 kubernetes.liveness_probe.failure.total,count,,,,Cumulative number of failed liveness probe for a container (ALPHA in kubernetes v1.15),-1,kubernetes,k8s.liveness_probe.failure.total,
 kubernetes.readiness_probe.success.total,count,,,,Cumulative number of successful readiness probe for a container (ALPHA in kubernetes v1.15),-1,kubernetes,k8s.liveness_probe.success.total,
 kubernetes.readiness_probe.failure.total,count,,,,Cumulative number of failed readiness probe for a container (ALPHA in kubernetes v1.15),-1,kubernetes,k8s.liveness_probe.failure.total,
+kubernetes.startup_probe.success.total,count,,,,Cumulative number of successful startup probe for a container (ALPHA in kubernetes v1.15),-1,kubernetes,k8s.startup_probe.success.total,
```

Comment:
As a result of [this PR](https://github.com/DataDog/integrations-core/pull/15167), the metric type of the probe metrics changed from `count` to `gauge`.

Reply:
I fixed the merge conflict and updated the metrics type